Cargo.toml

```diff
@@ -98,6 +99,9 @@ tokio = { version = "1.36.0", features = ["fs", "time"] }
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
 
+[target.'cfg(linux)'.build-dependencies]
+zbus = "3.15.2"
```

Comment:
This dependency is not used right now, did you plan to use it for something?

Reply:
Well, if you need to bump the version of the crate you don't need to add it to `Cargo.toml`, you can just update that particular crate with `cargo update -p crate_name`